docs/docs/09-faqs.md

```diff
@@ -53,3 +53,8 @@ This tells the LLM (by way of a [system message](https://platform.openai.com/doc
 
 This context also automatically shares the `sys.ls`, `sys.read`, and `sys.write` tools with the tool that is using it as a context. This is because if a tool intends to interact with the workspace, it minimally needs these tools.
 
+### What do I need to get gptscript working on windows?
```

Comment:
Just some nits on capitalization - I believe we call the tool `GPTScript`

Reply:
Addressed the suggested change.